src/server/test/util/unitTestUtils.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const { chai, mocha, expect, app} = require('../common');
+const chaiHttp = require('chai-http');
+chai.use(chaiHttp);
+const chaiRequest = require('chai').request;
+// SHL: Please remove blank line between imports.
+
+const Unit = require('../../models/Unit')
+const { getUnitId } = require('../../util/readingsUtils');
+
+const validPayload = {
+	name: 'Water Flow Unit',
+	identifier: 'energy_unit',
+	unitRepresent: Unit.unitRepresentType.FLOW,
+	secInRate: 60,
+	typeOfUnit: Unit.unitType.UNIT,
+	suffix: 'L/s',
+	displayable: Unit.displayableType.ALL,
+	preferredDisplay: true,
+	note: 'Used to measure water flow',
+	minVal: 1,
+	maxVal: 100,
+	disableChecks: Unit.disableChecksType.REJECT_BAD
+};
+
+/**
+ * Applies a mutation to a base object for testing.
+ * @param {Object} base - The base valid object.
+ * @param {Object} [mutation={}] - Mutation descriptor.
+ * @returns {Object} The mutated object.
+ */
+function applyMutation(base, mutation = {}) {
```

Comment:
I cannot find where this function is used. Given this and the comments below, I'm wondering about the entire file. Am I missing something?